mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
```

Comment:
Inconsistent error handling. In other parts of the codebase, error responses are handled with `errorHandler` function, but here it's missing. It should be consistent throughout the codebase.

Missing error message. The JSON object being returned should contain a meaningful error message to help with debugging and error handling.

Inconsistent status code. The status code 404 is being used, but in other parts of the codebase, 400 is used for similar error scenarios. It should be consistent throughout the codebase.